validator/sawtooth_validator/server/core.py

```diff
@@ -152,10 +153,22 @@ def __init__(self, network_endpoint, component_endpoint, public_uri,
                                      heartbeat=False,
                                      max_incoming_connections=20)
 
+        config_file = os.path.join(config_dir, "validator.toml")
```

Comment:
Do we need to create a task in JIRA to mirror the rest of the validator CLI arguments in this file? @vaporos Everything else looks good.